### ISSUE TITLE
Output format "table" on instance name result shows "NocaseDict" in keybindings

### ISSUE THREAD

**Issue by andy-maier** [pywbem/pywbemtools#258]
Example:
```
$ pywbemcli -m tests/unit/simple_mock_model_ext.mof -o table instance enumerate CIM_Foo -o
InstanceNames: CIM_Foo
+--------+-------------+------------------------------------------------+
| host   | namespace   | keybindings                                    |
|--------+-------------+------------------------------------------------|
|        | root/cimv2  | NocaseDict({'InstanceID': 'CIM_Foo1'})         |
|        | root/cimv2  | NocaseDict({'InstanceID': 'CIM_Foo2'})         |
|        | root/cimv2  | NocaseDict({'InstanceID': 'CIM_Foo3'})         |
|        | root/cimv2  | NocaseDict({'InstanceID': 'CIM_Foo30'})        |
|        | root/cimv2  | NocaseDict({'InstanceID': 'CIM_Foo31'})        |
|        | root/cimv2  | NocaseDict({'InstanceID': 'CIM_Foo_sub1'})     |
|        | root/cimv2  | NocaseDict({'InstanceID': 'CIM_Foo_sub2'})     |
|        | root/cimv2  | NocaseDict({'InstanceID': 'CIM_Foo_sub3'})     |
|        | root/cimv2  | NocaseDict({'InstanceID': 'CIM_Foo_sub_sub1'}) |
|        | root/cimv2  | NocaseDict({'InstanceID': 'CIM_Foo_sub_sub2'}) |
|        | root/cimv2  | NocaseDict({'InstanceID': 'CIM_Foo_sub_sub3'}) |
+--------+-------------+------------------------------------------------+
```

I suggest to manually remove "NocaseDict(" and ")" from the repr() result string. That leaves the order of keybindings unchanged.

Note that converting it to a dict would make the order of items arbitrary.

**Comment by KSchopmeyer** [comment]
How about this for the format of the table form of instance path:

```

InstanceNames: CIM_Foo
+--------+-------------+-----------------+-----------------------------+
| host   | namespace   | class           | keys                        |
|--------+-------------+-----------------+-----------------------------|
|        | root/cimv2  | CIM_Foo         | InstanceID:CIM_Foo1         |
|        | root/cimv2  | CIM_Foo         | InstanceID:CIM_Foo2         |
|        | root/cimv2  | CIM_Foo         | InstanceID:CIM_Foo3         |
|        | root/cimv2  | CIM_Foo         | InstanceID:CIM_Foo30        |
|        | root/cimv2  | CIM_Foo         | InstanceID:CIM_Foo31        |
|        | root/cimv2  | CIM_Foo_sub     | InstanceID:CIM_Foo_sub1     |
|        | root/cimv2  | CIM_Foo_sub     | InstanceID:CIM_Foo_sub2     |
|        | root/cimv2  | CIM_Foo_sub     | InstanceID:CIM_Foo_sub3     |
|        | root/cimv2  | CIM_Foo_sub_sub | InstanceID:CIM_Foo_sub_sub1 |
|        | root/cimv2  | CIM_Foo_sub_sub | InstanceID:CIM_Foo_sub_sub2 |
|        | root/cimv2  | CIM_Foo_sub_sub | InstanceID:CIM_Foo_sub_sub3 |
+--------+-------------+-----------------+-----------------------------+

**Comment by andy-maier** [comment]
I would definitely like to see one instance per output row, independent of the string length of the keys. The tables will automatically become as wide as needed.

**Comment by andy-maier** [comment]
Addressed in PR #259
